### PR TITLE
Feature/swagger docs

### PR DESCRIPTION
**Pull request by hb308**
# Description
Swagger docs have been added as a separate service. The swagger docs have been separated for each microservice but are combined into one service. Configuration has been done for both profile and activity APIs but auth needs to be done.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Jest testing the routes
- [x] Loading up the swagger docs correctly

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: activity-recorder/src/routes/index.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Activity } from '../models/activity';
 const router = express.Router();
 
 router.get('/api/activity', currentUser, requireAuth, async (req: Request, res: Response) => {
-    const usersActivities = await Activity.find({ userId: req.currentUser?.id });
+    const usersActivities = await Activity.find({ userId: req.currentUser!.id });
```
Comment (ndeasy) [inline activity-recorder/src/routes/index.ts:8]: Is this part of the swagger docs? Looks like a bug fix

File: profile/src/app.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { NotFoundError } from '@amp-rehab-app/common';
+import { NotFoundError, errorHandler } from '@amp-rehab-app/common';
```
Comment (ndeasy) [inline profile/src/app.ts:1]: Should this be in a separate ticket?

File: profile/src/routes/create-profile.ts [hunk 3]
```diff
@@ -16,14 +16,16 @@ router.post(
...
     validateRequest,
     async (req: Request, res: Response) => {
         const { isMilitary, branch, serviceId, email, fullName, displayName, profilePic, bio } = req.body;
 
-        const existingProfile = await Profile.find({ email: email})
+        const existingProfile = await Profile.findOne({ email: req.currentUser!.email});
 
         if(existingProfile){
             res.status(409);
+            res.send("Profile with this emaila already exists")
```
Comment (ndeasy) [inline profile/src/routes/create-profile.ts:28]: Spelling in "emaila". Not sure what the approach to strings is on the backend, but on mobile we store our strings in one place to make internationalisation easier.

File: profile/src/routes/create-profile.ts [hunk 4]
```diff
@@ -16,14 +16,16 @@ router.post(
         body('displayName').not().isEmpty().withMessage('You need to provide your display name'),
     ],
     currentUser,
+    requireAuth,
     validateRequest,
     async (req: Request, res: Response) => {
         const { isMilitary, branch, serviceId, email, fullName, displayName, profilePic, bio } = req.body;
 
-        const existingProfile = await Profile.find({ email: email})
+        const existingProfile = await Profile.findOne({ email: req.currentUser!.email});
```
Comment (ndeasy) [inline profile/src/routes/create-profile.ts:24]: Should this be in a separate ticket?

Comment (hb308) [reply profile/src/routes/create-profile.ts]: Thanks for the comments Niall, it was some minor spelling bugs I found as I was using the swagger api but I'll create a separate branch for the bug fixes :)

**Pull request merged**